Apromore-Core-Components/Apromore-Portal/src/main/webapp/index.zul

```diff
@@ -203,6 +203,10 @@
                             <button id="btnUpload" sclass="ap-icon ap-icon-upload" ca:data-t="upload" tooltiptext="${labels.e.portal.upload.hint}"/>
                             <button id="btnDownload" sclass="ap-icon ap-icon-download" ca:data-t="download" tooltiptext="${labels.e.portal.download.hint}"/>
                             <h:span class="ap-action-sep"></h:span>
```

Comment:
I'd be inclined to include the ETL as part of the same section that includes upload and download and not have the separator between them.  I don't feel strongly about it however, so I'll leave that potential change to your discretion.